src/bulkExport/bulkExportResults.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import AWS from 'aws-sdk';
```

Comment:
Should this import from `../../AWS`? I believe that allows us to track AWS calls using XRay?

Reply:
Are we not using `../AWS` because we're using the `SIGNER_ROLE_ARN`?

Reply:
I switched to using `../AWS`. The signer role does not interfere with x ray tracing. It is just a a parameter to a call to STS.

Actually this change is not required to make x-ray tracing work. It looks like 
```js
import AWS from 'aws-sdk';
const AWSWithXray = AWSXRay.captureAWS(AWS);
``` 
actually modifies the `AWS` config/prototype, so you don't really need to pass around the `AWSWithXray` result. Other modules can just `import AWS from 'aws-sdk';` and they'll have x-ray enabled. This is not obvious when looking at the docs of how to enable xray in nodejs.

This means that there's a less intrusive way to enable x-ray for an entire package. Still, updated it to be consistent for now. 

https://github.com/aws/aws-xray-sdk-node/blob/fdbbdf7b7e788a70d23d25637296b254a6171a2f/packages/core/lib/patchers/aws_p.js#L37